src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2853,14 +2853,21 @@ void ApplicationManagerImpl::UnregisterAllApplications() {
 
   hmi_cooperating_ = false;
   bool is_ignition_off = false;
-  using namespace mobile_api::AppInterfaceUnregisteredReason;
```

Comment:
the new code is equivalent to the old code, what's the reason behind this change (not using name space in this file)?

Reply:
@yang1070 You are right. The code is equal to the old one regarding functionality. But to avoid namespace pollution and to improve readability of code clear usage of entities form namespace were added. 
`mobile_api::AppInterfaceUnregisteredReason::INVALID_ENUM` is more clear than just `INVALID_ENUM`, isn't it?